website/sidebars.json

```diff
@@ -37,6 +37,7 @@
       "en/packages/fast-navigation-generator-react/index",
       "en/packages/fast-permutator/index",
       "en/packages/fast-sketch-library/index",
+      "en/packages/fast-tooling-react/index",
```

Comment:
same question with this file, is it being auto-generated?

Reply:
this file is not autogenerated.